apps/user/src/app/application/page.tsx

```diff
@@ -0,0 +1,118 @@
+'use client';
+
+import ApplicationInfo from '@/components/application/applicationInfo/ApplicationInfo';
+import { ROUTES } from '@/constants/common/constant';
+import { AppLayout } from '@/layouts';
+import { color } from '@maru/theme';
+import { Text, UnderLineButton } from '@maru/ui';
+import { flex } from '@maru/utils';
+import { usePathname, useRouter } from 'next/navigation';
+import styled from 'styled-components';
+
+const NAVIGATION_DATA = [
+    {
+        name: '학생, 학부모',
+        route: ROUTES.APPLICATION_STUDENT
+    },
+    {
+        name: '교사',
+        route: ROUTES.APPLICATION_TEACHER
+    }
+]
+
+const INFORMATION_DATA = [
+    {
+        date: '9월 16일(토) 11:00',
+        place: '본교 SRC관 1층',
+        deadline: '2023.09.01 ~ 2023.09.13',
+        applicable: true,
+        statusText: '신청 가능'
+    },
+    {
+        date: '10월 4일 (수) 19:00',
+        place: '본교 SRC관 1층',
+        deadline: '2023.09.11 ~ 2023.09.25',
+        applicable: true,
+        statusText: '신청 가능'
+    },
+    {
+        date: '7월 8일 (토) 11:00',
+        place: '본교 SRC관 1층',
+        deadline: '2023.06.25 ~ 2023.07.02',
+        applicable: false,
+        statusText: '마감'
+    },
+    {
+        date: '8월 26일(토) 11:00',
+        place: '본교 SRC관 1층',
+        deadline: '2023.08.09 ~ 2023.08.23',
+        applicable: false,
+        statusText: '조기 마감'
+    },
+]
+
+const ApplicationPage = () => {
+    const router = useRouter();
+    const pathName = usePathname();
+
+    return (
+        <AppLayout header footer style={{ padding: '0px 312px', marginTop: 82 }}>
```

Comment:
```suggestion
        <AppLayout header footer>
```

저 스타일은 StyledApplicationPage 줘도 동일하게 적용될것같아요 !